src/ssb-server.js

```diff
@@ -28,8 +29,19 @@ module.exports = function SSB (opts = {}) {
 
     .use(require('ssb-friends'))
     .use(require('ssb-lan'))
-    .use(require('ssb-conn'))
     .use(require('ssb-ebt'))
+    // .use(require('ssb-conn'))
```

Comment:
TODO: we see that turning off the default scheduler works amazingly for initial sync,
but not sure what the implication is long term. I think we need to make our own scheduler to make sure we have some connections live, but not too many, and cycle through them